recipy/log.py

```diff
@@ -67,7 +67,8 @@ def log_init():
         "date": datetime.datetime.utcnow(),
         "command_args": " ".join(cmd_args),
         "warnings": [],
-        "libraries": [get_version('recipy')]
+        "libraries": [get_version('recipy')],
+        "custom-values": {}
```

Comment:
Very minor thing: other multiple-word keys here are split by `_` not `-` - could you alter here (and below) to be consistent with this?